src/types/spend_bundle.py

```diff
@@ -46,6 +46,12 @@ def announcements(self) -> List[Announcement]:
             items.extend(coin_solution.announcements())
         return items
 
+    def announcement_names(self) -> List[bytes32]:
```

Comment:
return [cs.announcement_names() for cs in self.coin_solutions]

Reply:
Implemented